dartmouth-outing-club/trailhead
# Convert table names to snake_case

## Description
Postgres expects lowercase table names; it will automatically convert the table names in your commands to lowercase unless you explicitly quote them.

Resolves: #66

## Testing
1. Reset postgres
   * Delete existing postgres database
   * `initdb -U postgres postgres`
   * `postgres -D postgres` (or however you prefer to start postgres)
   * `createdb -U postgres trailhead_development`
2. Inside the server directory, run: `yarn sequelize db:migrate && yarn sequelize db:seed:all`
3. Login to the postgres console with `psql -U postgres trailhead_development`
4. View the tables with `\dt`. 
5. Query a table and verify that the entries were successfully seeded i.e. `select * from people;`
